conda-recipe/meta.yaml

```diff
@@ -17,7 +17,9 @@ requirements:
     host:
         - python
         - setuptools >=63.*
-        - numba 0.57*
+        - scikit-build
+        - cmake
```

Comment:
Do not we need a version check here for cmake? In the `CMakeLists.txt` we have `
cmake_minimum_required(VERSION 3.21...3.27 FATAL_ERROR)`

Reply:
Fixed.